plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/inspections/UnusedUnaryOperatorInspection.kt

```diff
@@ -39,31 +40,30 @@ class UnusedUnaryOperatorInspection : AbstractKotlinInspection() {
         val operationToken = prefix.operationToken
         if (operationToken != KtTokens.PLUS && operationToken != KtTokens.MINUS) return
 
-        // hack to fix KTIJ-196 (unstable `USED_AS_EXPRESSION` marker for KtAnnotationEntry)
-        if (prefix.isInAnnotationEntry) return
```

Comment:
This line is still needed, even though tests pass without it :( I will expand the comment.